maintainers/maintainer-list.nix

```diff
@@ -6890,6 +6890,11 @@
     githubId = 1595132;
     name = "Kranium Gikos Mendoza";
   };
+  wooosh = {
+    email = "crimsonwombat42@gmail.com";
+    github = "wooosh";
+    name = "wooosh";
```

Comment:
Can you add your `githubId` too please?

You can find it here: https://api.github.com/users/wooosh .

Basically, that's because usernames on GH can change, but ids won't.

See details of why it's useful to NixOS/nixpkgs: https://github.com/NixOS/rfcs/blob/master/rfcs/0039-unprivileged-maintainer-teams.md#changes-to-maintainersmaintainer-listnix .